docs/docusaurus/docs/conceptual_guides/expectation_classes.md

```diff
@@ -0,0 +1,124 @@
+
+GX doesn’t have a TableExpectation type because you can get the same functionality from a BatchExpectation. If you want to run Expectations on an entire table, you configure a DataAsset to use an entire table as its domain.
+
+![BatchExpectation](../images/batch_expectation.png)
+
+## Aggregate Expectations
+
+Aggregate Expectations are based on a single Metric for the whole Batch. This Metric is called the `observed_value` for the Expectation. 
+
+A common pattern is to calculate a numeric Metric, and then verify that it falls between a `min_value` and `max_value`, as in `expect_column_mean_to_be_between`. However, some Expectations only have a max or a min, such as `expect_column_kl_divergence_to_be_less_than`. 
+
+Some Expectations don’t use a numeric Metric for the observed_value. For example, [`expect_column_distinct_values_to_equal_set`](https://greatexpectations.io/expectations/expect_column_distinct_values_to_equal_set?filterType=Backend%20support&gotoPage=1&showFilters=true&viewType=Summary) creates a set of distinct column values, that is then compared against a specified set and [`expect_column_to_have_no_days_missing`](https://greatexpectations.io/expectations/expect_column_distinct_values_to_equal_set?filterType=Backend%20support&gotoPage=1&showFilters=true&viewType=Summary) looks for continuity within the column’s values.
+
+![Aggregate Expectations](../images/aggregate_expectations.png)
+
+Aggregate Expectations calculate summary statistics across Batches of data. As a result, they can be a computationally efficient way to gain insight into the overall behavior of a dataset and can provide a useful foundation for identifying trends, patterns, and outliers. However, because Aggregate Expectations do not verify individual rows of data, they can't identify specific data issues.
+
+## Map Expectations
+
+Map Expectations are evaluated on a row-by-row basis and each row is checked independently. For example, [`expect_column_values_to_not_be_null`](https://greatexpectations.io/expectations/expect_column_values_to_not_be_null?filterType=Backend%20support&gotoPage=1&showFilters=true&viewType=Summary), [`expect_column_values_to_be_in_set`](https://greatexpectations.io/expectations/expect_column_values_to_not_be_null?filterType=Backend%20support&gotoPage=1&showFilters=true&viewType=Summary), [` expect_column_pair_values_to_be_equal`](https://greatexpectations.io/expectations/expect_column_pair_values_to_be_equal?filterType=Backend%20support&gotoPage=1&showFilters=true&viewType=Summary).
+
+![Map Expectations](../images/map_expectations.png)
+
+Map Expectations are useful when you want to be certain that the content of a given dataset is correct. If you’re validating data within a pipeline, Map Expectations can help you identify invalid rows, remove invalid rows from the dataset, and process the remaining data. Unfortunately, Map Expectations evaluate every row of data and this can be computationally intensive.
```

Comment:
```suggestion
Map Expectations are useful when you want to be certain that the content of a given dataset is correct. If you’re validating data within a pipeline, Map Expectations can help you identify invalid rows, remove invalid rows from the dataset, and process the remaining data. Unfortunately, because Map Expectations evaluate every row of data they can be computationally intensive.
```